src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.GenerateAssemblyInfo.targets

```diff
@@ -32,6 +32,11 @@ Copyright (c) .NET Foundation. All rights reserved.
       _CreateRazorTargetAssemblyInfoInputsCacheFile;
       CoreGenerateRazorTargetAssemblyInfo
     </GenerateRazorTargetAssemblyInfoDependsOn>
+
+    <GenerateRazorTargetAssemblyInfoDependsOn Condition="'$(_Targeting30OrNewerRazorLangVersion)' == 'true'">
```

Comment:
Is this the right condition? Was this functionality in Razor.Design before?

Reply:
These properties were previously declared by Mvc.Razor.Extensions. This target file is imported for both 2.x and 3.x while the new target is only available in 3.x. I considered moving this property assignment inside Razor.Configuration.targets, but this seemed like it would be easier to follow what's happening.